sys/fs/fuse/fuse_vnops.c

```diff
@@ -2744,17 +2745,26 @@ fuse_vnop_setextattr(struct vop_setextattr_args *ap)
 	len = strlen(prefix) + sizeof(extattr_namespace_separator) +
 	    strlen(ap->a_name) + 1;
 
-	fdisp_init(&fdi, len + sizeof(*set_xattr_in) + uio->uio_resid);
+	/* we want to be binary compatible */
```

Comment:
"binary compatible with what"?  I suggest this wording instead.
```suggestion
	/* older FUSE servers  use a smaller fuse_setxattr_in struct*/
```